recipes/libnice/all/test_package/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conan import ConanFile
+from conan.tools.meson import Meson
+from conan.tools.layout import basic_layout
+from conan.tools.build import cross_building
+
+
+class LibniceTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "PkgConfigDeps", "MesonToolchain"
+
```

Comment:
```suggestion
    test_type = "explicit"

    def requirements(self):
        self.requires(tested_reference_str)
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe